### DISCUSSION TITLE
%meth difference for defining DML with methylKit

### DISCUSSION THREAD

**Discussion by yaaminiv** [RobertsLab/resources#1140] [Q&A]
In my Cv paper, I defined DML as CpGs that are 50% different between treatments and q-value < 0.01 identified by `methylKit`. I've seen some vertebrate papers that use a 10% or 25% difference between treatments to define DML (ex. Mac's salmon paper). I believe Claire also used a 25% difference in `methylKit` (doi.org/10.1101/012831). 

Should I use a 25% difference instead of 50%? I could easily try both and see what happens, but I want to know if there's a clearer rationale for picking 25% vs. 50%.

**Comment by sr320** [comment]
I would try both, no clear rationale as we know the biological signficance
of neither.

>
